types/tedious/v4/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Also i dont think you need v4 copy at all.

Reply:
Ok, I've reverted the change now